Makefile

```diff
@@ -47,11 +47,11 @@ dist: distclean
 		$(PYTHON) setup.py sdist; \
 		$(PYTHON) setup.py bdist_wheel --universal
 dist-upload: dist-check-version clean test dist-upload-twine
-dist-check-version: PKG_VER=v$(shell sed -n "s/^.*VERSION\s\+=\s\+'\([^']\+\)'.*$$/\1/p" setup.py)
+dist-check-version: PKG_VER=v$(shell sed -n "s/^.*__version__\s\+=\s\+'\([^']\+\)'.*$$/\1/p" learnosity_sdk/_version.py)
```

Comment:
You should be able to use `\s` to match any whitespace (space, tab, ...) (thought my `\+` in the initial line was misguided, and `*` is indeed better).

Also, I'd rather not be too specific on the format of the version, we just want to extract anything that's between the two quotes, which is what my `\([^']\+\)` was doing.

Reply:
@shtrom were you commenting on an earlier fixup where I had done this? (To make the sed regex run on macOS)

```
sed -n "s/^__version__[ ]*=[ ]*'\([0-9a-zA-Z\.]*\)'$$/\1/p" learnosity_sdk/_version.py
```

I've since changed it back to something closer to the original regex. As it's probably more robust if it only works on linux.

Reply:
Ah, yeah, I did, and was confused by a fixup, so I just copy/refresh/pasted my comment.

Reply:
You can try to run a `make dist-check-version`, and make sure that the version is properly extracted from the version file by looking at the command it runs.

Reply:
I'd like to 2nd using `*` instead of `\+` here just to account for the event where a space is somehow accidentally removed.